website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -14,6 +14,16 @@ provided for their upgrades as a result of new features or changed behavior.
 This page is used to document those details separately from the standard
 upgrade flow.
 
+## Consul 1.16.x
+
+#### Remove deprecated service-defaults peer upstream override behavior
+
+In Consul 1.16.x, `service-defaults` upstream [`overrides`](/consul/docs/connect/config-entries/service-defaults#overrides)
+will not apply to peer upstreams, unless the [`peer`](/consul/docs/connect/config-entries/service-defaults#peer) field is explicitly provided.
+
+Consul 1.15.x introduced a backward-compatibility behavior to allow for a smoother transition during the upgrade. This behavior will be removed in Consul 1.16.x.
+Visit the [upgrade instructions for 1.15.x](#service-defaults-overrides-for-upstream-peered-services) for more info.
```

Comment:
```suggestion
The `service-defaults` upstream [`overrides`](/consul/docs/connect/config-entries/service-defaults#overrides)
parameter does not apply to peer upstreams unless the [`peer`](/consul/docs/connect/config-entries/service-defaults#peer) field is explicitly provided.

This change removes the backward-compatibility behavior introduced in Consul 1.15.x. Refer to the [upgrade instructions for 1.15.x](#service-defaults-overrides-for-upstream-peered-services) for more information.
```
Switch to present tense per the style guide. Will there be a separate PR for updating the configuration entry reference?

Reply:
Sorry about that. Here's the docs PR. https://github.com/hashicorp/consul/pull/16960/files